### PR TITLE
Addition of Elven Containers and Missing ML Artifacts

### PR DESCRIPTION
**Pull request by Arthrutus**
Added support for ML Artifacts "Mischief Maker" and "Faerie Fire" with stats according to https://uo.com/wiki/ultima-online-wiki/items/artifact-collections/artifacts-craftable-major/

Also added support for ML elven containers (Arcane bookshelf, Ornate Elven Chest, Elven Dress, and Elven Wash Basin [As well as East/South]). Please note that the east facing arcane bookshelf is labeled elven bookshelf. This is a bug by OSI that was never fixed and still exists in the game today. A simple fix for those who prefer it patched is to add an override to force a labelnumber corresponding to a cliloc in UOFiddler.

Support for the Fancy Elven Armoire was added in. All items are also craftable via the carpentry gump menu.

Also cleaned up whitespace in DefCarpentry.cs to present a more organized look.

### CODE REVIEW

File: Projects/UOContent/Items/Addons/ElvenForgeAddon.cs [hunk 1]
```diff
@@ -21,6 +21,6 @@ public ElvenForgeDeed()
         }
 
         public override BaseAddon Addon => new ElvenForgeAddon();
-        public override int LabelNumber => 1072875; // squirrel statue (east)
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Addons/ElvenForgeAddon.cs:24]: WTF? LOL Crazy localization comment.

File: Projects/UOContent/Items/Addons/SpinningwheelEastAddon.cs [hunk 2]
```diff
@@ -2,14 +2,6 @@
 
 namespace Server.Items
 {
-    public delegate void SpinCallback(ISpinningWheel sender, Mobile from, int hue);
-
-    public interface ISpinningWheel
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Addons/SpinningwheelEastAddon.cs:7]: I am not sure this should have been moved. Let's move it back?

File: Projects/UOContent/Items/Construction/Chairs/Chairs.cs [hunk 3]
```diff
@@ -171,14 +171,14 @@ public override void Deserialize(IGenericReader reader)
     }
 
     [DynamicFliping, Flippable(0x2DEB, 0x2DEC, 0x2DED, 0x2DEE)]
-    public class BigElvenChair : Item
+    public class CozyElvenChair : Item
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Construction/Chairs/Chairs.cs:174]: Let's add a TypeAlias for BigElvenChair so we don't break people's saves.

File: Projects/UOContent/Items/Construction/Chairs/Chairs.cs [hunk 4]
```diff
@@ -198,14 +198,14 @@ public override void Deserialize(IGenericReader reader)
     }
 
     [DynamicFliping, Flippable(0x2DF5, 0x2DF6)]
-    public class ElvenReadingChair : Item
+    public class ReadingChair : Item
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Construction/Chairs/Chairs.cs:201]: Same thing with this one.

File: Projects/UOContent/Items/Construction/Decorative/GiantReplicaAcorn.cs [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        {
+        }
+
+        public override int LabelNumber => 1072889; // giant replica acorn
+
+        public override void Serialize(IGenericWriter writer)
+        {
+            base.Serialize(writer);
+
+            writer.Write(0); // version
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Construction/Decorative/GiantReplicaAcorn.cs:19]: Let's codegen new stuff if possible.

File: Projects/UOContent/Items/Addons/SpinningwheelEastAddon.cs (hunk 2 above)
Comment (Arthrutus) [reply Projects/UOContent/Items/Addons/SpinningwheelEastAddon.cs:7]: it is moved to elvenspinningwheelEastaddon

Comment (Arthrutus) [reply Projects/UOContent/Items/Addons/SpinningwheelEastAddon.cs:7]: This was moved over to ElvenSpinningWheelEastAddon.cs

File: Projects/UOContent/Items/Construction/Chairs/Chairs.cs (hunk 3 above)
Comment (Arthrutus) [reply Projects/UOContent/Items/Construction/Chairs/Chairs.cs]: Reverted back as name change was unneeded! Still shows proper name.

File: Projects/UOContent/Items/Construction/Chairs/Chairs.cs (hunk 4 above)
Comment (Arthrutus) [reply Projects/UOContent/Items/Construction/Chairs/Chairs.cs]: Same with this one!

File: Projects/UOContent/Items/Addons/ElvenForgeAddon.cs (hunk 1 above)
Comment (Arthrutus) [reply Projects/UOContent/Items/Addons/ElvenForgeAddon.cs:24]: This isn't accurate. I guess my push didnt push LOL

File: Projects/UOContent/Items/Construction/Decorative/GiantReplicaAcorn.cs (hunk 5 above)
Comment (Arthrutus) [reply Projects/UOContent/Items/Construction/Decorative/GiantReplicaAcorn.cs:19]: This works as intended and will create the item from the carpentry menu :D

File: Projects/UOContent/Engines/Craft/DefBlacksmithy.cs [hunk 6]
```diff
@@ -55,9 +55,9 @@ public static void CheckAnvilAndForge(Mobile from, int range, out bool anvil, ou
                 var type = item.GetType();
 
                 var isAnvil = type.IsDefined(typeofAnvil, false) || item.ItemID == 4015 || item.ItemID == 4016 ||
-                              item.ItemID == 0x2DD5 || item.ItemID == 0x2DD6;
+                              item.ItemID == 11733 || item.ItemID == 11734;
```
Comment (kamronbatman) [inline Projects/UOContent/Engines/Craft/DefBlacksmithy.cs:58]: @Arthrutus any particular reason to change these to decimal?

**Comment by kamronbatman** [comment]
I am going to pull this into a feature branch so it isn't from your fork's `main` branch. That way it isn't subject to accidental commits. Then we can iterate on it.
Will probably require closing this PR and making a new one.

File: Projects/UOContent/Items/Addons/ElvenSpinningwheelEastAddon.cs [hunk 7]
```diff
@@ -93,7 +98,7 @@ protected override void OnTick()
         }
     }
 
-    [Serializable(0)]
+    [Serializable(0, false)]
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Addons/ElvenSpinningwheelEastAddon.cs:101]: These shouldn't be changed, in RunUO they were encoded,

**Pull request closed without merging**